mailbox/plugin/quota-search-elasticsearch/src/main/java/org/apache/james/quota/search/elasticsearch/QuotaQueryConverter.java

```diff
@@ -66,6 +66,10 @@ public QueryBuilder from(QuotaQuery query) {
             return toElasticSearch(clauses.get(0));
         }
         
+        return asBoolQueryBuilder(clauses);
```

Comment:
given the beginning of the method, I would have call it toElasticSearch.
And thinking about it, I would rather rename toElasticSearch to singleClauseAsESQuery and the second one clausesAsAndESQuery (or anything that would expose the fact we compose clauses with a `and`)